test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/SortTests.cs

```diff
@@ -193,6 +193,23 @@ public async Task GivenPatients_WhenSearchedWithFamilySortParams_ThenPatientsAre
             SortTestsAssert.AssertPatientFamilyNamesAreEqualInRange(0, patients.OrderBy(x => x.Name.Min(n => n.Family)).ToList(), returnedResults);
         }
 
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenPatients_WhenSearchedWithTextSortAndInclude_ThenPatientsAreReturned()
+        {
+            var tag = Guid.NewGuid().ToString();
+            var patients = await CreatePatients(tag);
+            Assert.True(patients.Count() > 3);
+
+            // let's make sure that nothing is broken without sort first. This also tests pagination.
+            var results = await GetResultsFromAllPagesAsync($"Patient?_count=3&_include=Observation:performer");
```

Comment:
Should the _tag filter be included here? With only 3 per page this could take some time on the persistent databases.

Reply:
Fixed